src/scenes/Dashboard/components/TopActions/index.js

```diff
@@ -33,24 +38,32 @@ class TopActions extends Component {
     super(props);
 
     this.onSortOptionSelected = this.onSortOptionSelected.bind(this);
+    this.onCreateDialogClose = this.onCreateDialogClose.bind(this);
+    this.onCreateDialogOpen = this.onCreateDialogOpen.bind(this);
```

Comment:
also, we should make these fat arrow functions, although I know you said you'd do that later :)